TEveEventDisplay/src/TEveEventDisplay_module.cc

```diff
@@ -100,11 +100,13 @@ namespace mu2e
       application_ = new TApplication( "noapplication", &tmp_argc, tmp_argv );
     } 
     //construct GUI:
-    _frame = new TEveMu2eMainWindow(gClient->GetRoot(), 1000,600, _pset);
-    //build 2D geometries:
-    _frame->CreateCaloProjection();
+    
+    DrawOptions DrawOpts(_filler.addCrvHits_, _filler.addCosmicSeedFit_, _filler.addTracks_, _filler.addClusters_, _filler.addHits_, false, _filler.addMCTraj_); 
+    _frame = new TEveMu2eMainWindow(gClient->GetRoot(), 1000,600, _pset, DrawOpts);
```

Comment:
Does root/Eve take ownership of this pointer and delete the pointer it at the end of the job?  Or are you responsible for deleting it at the end of the job?

Reply:
I think it deletes it

Reply:
I think it's a good idea to add a comment about that at the creation of every object that root will eventually delete for you.

A better idea is for us to write a trivial class template that, under the covers, is just a bare pointer.  The idea is that when you create an object that root will eventually take ownership of, then you make it this type of pointer.  This explains to future maintainers who is responsible for deleting pointers.  It also makes it easier to audit code to ensure that there are no memory leaks.

If I write this class template, are you willing to use it?

Reply:
I can attempt to use it if written, if ROOT/TEve doesnt complain, a lot of this is balancing goof practice to avoiding pitfalls in TEve